worlds/civ_6/Data.py

```diff
@@ -0,0 +1,79 @@
+from dataclasses import dataclass
+import json
+import os
+import pkgutil
+from typing import List
+
+
+_cache = {}
+
+
+def _get_data(key: str):
+    global _cache
+    if key not in _cache:
+        path = os.path.join("data", f"{key}.json")
+        _cache[key] = json.loads(
+            pkgutil.get_data(__name__, path).decode())
+    return _cache[key]
+
+
+def get_boosts_data():
+    return _get_data("boosts")

```

Comment:
```suggestion
```
You redefine this below, and it's very clear the one below is the one you actually want